salt/metalk8s/kubernetes/coredns/deployed.sls

```diff
@@ -6,111 +6,129 @@
 {% set context = "kubernetes-admin@kubernetes" %}
 
 Create coredns ConfigMap:
-  metalk8s_kubernetes.configmap_present:
-    - name: coredns
-    - namespace: kube-system
+  metalk8s_kubernetes.object_present:
```

Comment:
`salt/metalk8s/kubernetes/coredns/deployed/init.sls` but yes it's doable.

Just wondering if we want to do this because here with a state we have some `require`s, thinks we can not have with the renderer, but agree would be simpler to update/edit/whatever if we have a state in kubernetes yaml format instead of salt states

Reply:
#2026